### ISSUE TITLE
A lone newline is displayed when run_command returns an empty output_msg

### ISSUE THREAD

**Issue by crdx** [avantrec/soco-cli#17]
Thanks for all the work you've been putting into improving this tool recently.

I believe this bug was introduced in 0184bb3c3aabf6900d264359454caceb777dd86f.

The code that processes the result of `run_command` is now:

```py
if exit_code == 0:
    print(output_msg)
else:
    print(error_msg)
```

So if `output_msg` is empty then a blank line is produced on stdout because `print()` will always print a newline.

This may or may not affect `error_msg` in a similar way.

**Comment by pwt** [comment]
Good catch -- thanks for the detailed bug report.
Should be fixed in v0.3.23.

**Comment by crdx** [comment]
No problem. Thanks!
